di_website/common/blocks.py

```diff
@@ -3,28 +3,49 @@
 from django.core.exceptions import ValidationError
 from django.forms import Media
 from django.forms.utils import ErrorList
-
-from wagtail.core.blocks import (
-    BooleanBlock,
-    CharBlock,
-    ChoiceBlock,
-    ListBlock,
-    PageChooserBlock,
-    RichTextBlock,
-    StreamBlock,
-    StructBlock,
-    TextBlock,
-    URLBlock
-)
+from wagtail.core.blocks import (BooleanBlock, CharBlock, ChoiceBlock,
+                                 ListBlock, PageChooserBlock, RichTextBlock,
+                                 StreamBlock, StructBlock, TextBlock, URLBlock)
 from wagtail.documents.blocks import DocumentChooserBlock
 from wagtail.embeds.blocks import EmbedBlock
 from wagtail.images.blocks import ImageChooserBlock
 
-from di_website.common.constants import RICHTEXT_FEATURES, RICHTEXT_FEATURES_NO_FOOTNOTES
+from di_website.common.constants import (RICHTEXT_FEATURES,
+                                         RICHTEXT_FEATURES_NO_FOOTNOTES)
 from di_website.publications.blocks import AudioMediaBlock
 from di_website.publications.infographic import Infographic
 
 
+RED = 'poppy'
+BLUE = 'bluebell'
+PINK = 'rose'
+YELLOW = 'sunflower'
+ORANGE = 'marigold'
+PURPLE = 'lavendar'
+GREEN = 'leaf'
+COLOUR_CHOICES = (
+    (RED, 'Red'),
+    (BLUE, 'Blue'),
+    (PINK, 'Pink'),
+    (YELLOW, 'Yellow'),
+    (ORANGE, 'Orange'),
+    (PURPLE, 'Purple'),
+    (GREEN, 'Green')
+)
+class AccordionBlock(StructBlock):
+    colour = ChoiceBlock(choices=COLOUR_CHOICES, default=RED)
```

Comment:
This isn't the way to do it ... colours (themes) are tied to the entire page, and pages that support this kind of theming already have this colour field. So your accordion CSS should be based off those individual body colours e.g pages with the `rose` theme have the class `body--rose` added to the body element. So this is redundant as well.

<img width="659" alt="image" src="https://user-images.githubusercontent.com/5672438/193995683-2a744fed-8f85-44de-943c-cc6e4f21e74c.png">